StatusMonitor/StatusMonitor.cs

```diff
@@ -174,128 +168,103 @@ public List<Event> ParseStatusEntry(string line)
-                                            break;
-                                        }
-                                    case 1: // InternalPanel (right hand side)
-                                        {
-                                            status.gui_focus = "internal panel";
-                                            break;
-                                        }
-                                    case 2: // ExternalPanel (left hand side)
-                                        {
-                                            status.gui_focus = "external panel";
-                                            break;
-                                        }
-                                    case 3: // CommsPanel (top)
-                                        {
-                                            status.gui_focus = "communications panel";
-                                            break;
-                                        }
-                                    case 4: // RolePanel (bottom)
-                                        {
-                                            status.gui_focus = "role panel";
-                                            break;
-                                        }
-                                    case 5: // StationServices
-                                        {
-                                            status.gui_focus = "station services";
-                                            break;
-                                        }
-                                    case 6: // GalaxyMap
-                                        {
-                                            status.gui_focus = "galaxy map";
-                                            break;
-                                        }
-                                    case 7: // SystemMap
-                                        {
-                                            status.gui_focus = "system map";
-                                            break;
-                                        }
-                                }
-                                status.latitude = JsonParsing.getOptionalDecimal(data, "Latitude");
-                                status.longitude = JsonParsing.getOptionalDecimal(data, "Longitude");
-                                status.altitude = JsonParsing.getOptionalDecimal(data, "Altitude");
-                                status.heading = JsonParsing.getOptionalDecimal(data, "Heading");
-
-                                // Spin off a thread to pass status entry updates in the background
-                                Thread updateThread = new Thread(() => handleStatus(timestamp, status));
-                                updateThread.IsBackground = true;
-                                updateThread.Start();
+                                status.gui_focus = "none";
```

Comment:
I wonder if an enum would be better than a string for this property.

Reply:
We want the output to be easy for the end user to parse. I think an enum might be a little less user friendly?

Reply:
I’ve been spoiled by Swift’s enums, which can be backed by string as easily as by int. Oh well.